workflow-bot-app/src/main/java/com/symphony/bdk/workflow/api/v1/dto/VersionedWorkflowView.java

```diff
@@ -0,0 +1,36 @@
+package com.symphony.bdk.workflow.api.v1.dto;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class VersionedWorkflowView {
+  @JsonProperty(access = JsonProperty.Access.READ_ONLY)
+  private String id;
+
+  @JsonProperty(access = JsonProperty.Access.READ_ONLY)
+  private String workflowId;
+
+  @JsonProperty(access = JsonProperty.Access.READ_ONLY)
+  private Long version;
+
+  @JsonProperty(access = JsonProperty.Access.READ_ONLY)
+  private Boolean active;
+
+  @JsonProperty(access = JsonProperty.Access.READ_ONLY)
+  private Boolean published;
+
+  @JsonProperty(access = JsonProperty.Access.READ_ONLY)
+  private String deploymentId;
+
+  @JsonProperty(access = JsonProperty.Access.READ_ONLY)
+  private String lastUpdatedBy;
+
+  private String swadl;
```

Comment:
Can't we use SwadlView here instead of putting the 2 fields ?

Reply:
then it s going to have an additional level in json payload